batch/batch/server/server.py

```diff
@@ -483,7 +483,7 @@ def is_successful(self):
             terminated = pod.status.container_statuses[0].state.terminated
             self.exit_code = terminated.exit_code
             if terminated.finished_at is not None and terminated.started_at is not None:
-                self.duration = (terminated.finished_at - terminated.started_at).total_seconds()
+                self.duration += (terminated.finished_at - terminated.started_at).total_seconds()
             else:
```

Comment:
won't this fail since duration starts as None?

Reply:
fixed